nixos/modules/misc/ids.nix

```diff
@@ -221,6 +221,7 @@
       skydns = 197;
       ripple-rest = 198;
       nix-serve = 199;
+      tvheadend = 200;
```

Comment:
Don't you need extragroup? 
Anyway create it and comment with `#` if you don't need.